File: ILT.IHR.DTO/UserNotification.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ILT.IHR.DTO
{
	public class UserNotification : AbstractDataObject
	{
		public int NotificationID { get; set; }
		public string Module { get; set; }
		public int UserID { get; set; }
		public Boolean IsAck { get; set; }
		public DateTime? AckDate { get; set; }

	}
}

File: ILT.IHR.APP/ClientApp/src/app/core/interfaces/UserNotification.ts
export class UserNotification {
    NotificationID: number;
    Module?: string;
    UserID: number;
    IsAck: boolean;
    AckDate?: Date;
    CreatedDate: Date;
    CreatedBy: string;
    ModifiedDate: Date;
    ModifiedBy: string;
}
export class UserNotificationDisplay {
    notificationID: number;
    module?: string;
    userID: number;
    isAck: boolean;
    ackDate?: Date;
    createdDate: Date;
    createdBy: string;
    modifiedDate: Date;
    modifiedBy: string;
}
